tests/unit/routes/test_lazy_app_container.py

```diff
@@ -1,64 +1,17 @@
 """
-Test app container
+Test lazy app container module
```

Comment:
Curious whether we see reduction in loading time in the `tests.unit.test_app.test_get_app__loading_time` test? If yes, maybe we should update the test there (use lower limit).